app/controllers/pins_controller.rb

```diff
@@ -15,6 +15,27 @@ def show
     @pin = Pin.find(params[:id])
   end
 
+  def unlike
```

Comment:
These two methods should be in the LikesController, under create and destroy!
When you "like" a pin, you are in fact creating a Like instance (with the user_id and pin_id associations), and when you "unlike" you are destroying a Like instance